### PR TITLE
Documents flow to create new ocne on oci clusters in console ui

### PR DESCRIPTION
**Pull request by rhomast**

### CODE REVIEW

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+1. From the left menu, select **Clusters**, and then click **Create**.
+1. Select **Oracle OCNE on OCI** and provide a name for the cluster.
+1. Expand **Member Roles** to add any users that you want grant access to this cluster and their permissions.
+1. Expand **Labels and Annotations** to configure Kubernetes [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) and [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) for the cluster.
+1. Select the cloud credentials that you created. Ensure that the appropriate Region and Compartment are selected from their dropdown lists. 
+1. Click **Authenticate and Configure Cluster**.
+1. Set up your network. Choose **Quick Create** to build a new virtual cloud network (VCN) using default settings or **Existing Infrastructure** to use an already configured VCN. If you choose to use an existing VCN, make sure that it is configured to accept the [ports and protocols required by Kubernetes](https://kubernetes.io/docs/reference/networking/ports-and-protocols/).
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:30]: We removed this radio button, users will need to provide their VCN Details

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+weight: 1
+draft: false
+---
+
+### Before you begin
+
+You'll need:
+
+* An Oracle Cloud Infrastructure (OCI) account with a compartment configured
+* An SSH key pair to use for cluster authentication
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:13]: Also, you will need a VCN configured

File: content/en/docs/setup/provision-cluster/ocne-oci.md (hunk 1 above)
Comment (anders-swanson) [reply content/en/docs/setup/provision-cluster/ocne-oci.md:30]: We should also mention, there must be bi-directional communication from the admin cluster to the managed cluster

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+1. Expand **Labels and Annotations** to configure Kubernetes [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) and [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) for the cluster.
+1. Select the cloud credentials that you created. Ensure that the appropriate Region and Compartment are selected from their dropdown lists. 
+1. Click **Authenticate and Configure Cluster**.
+1. Set up your network. Choose **Quick Create** to build a new virtual cloud network (VCN) using default settings or **Existing Infrastructure** to use an already configured VCN. If you choose to use an existing VCN, make sure that it is configured to accept the [ports and protocols required by Kubernetes](https://kubernetes.io/docs/reference/networking/ports-and-protocols/).
+1. Click **Configure Control plane and worker nodes**.
+1. Choose a Node Image from the dropdown list.
+1. Expand **OCNE Image Configuration** if you want to modify the default settings. You can add an optional custom image OCID to override the default settings of the node image and you can block the installation of OCNE dependencies on each node.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:33]: You may override the Node Image by providing a custom image OCID. If your custom image includes OCNE binaries, you may also skip OCNE dependency installation (advanced users).

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+1. Copy or upload an SSH public key to manage authentication of the cluster.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:34]: Your SSH public key will be installed on cluster nodes, enabling SSH after the cluster is created.

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+1. Configure the Control Plane. Select the **OCNE Version** first as it determines which Kubernetes versions are available, then choose a **Kubernetes Version** and a **Control Plane Shape**. You can leave the rest of the options at their default setting or modify them as needed.
+    
+    Under **Advanced**, you can choose to edit image tags for **ETCD**, **CoreDNS**, and **Calico**, or whether to install **OCI CCM/CSI** and **Calico**.
+1. Configure Node Pools, if necessary for your environment.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:38]: Add node pools for your cluster. Clusters without node pools will allow pods to schedule on the control plane.

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+1. Install Verrazzano on the cluster. Choose a **Verrazzano version** from the dropdown list. You can also expand **Advanced** to make changes to the Verrazzano Resource YAML. By default, Verrazzano is installed using the `managed-cluster` profile which enables a limited set of components on the cluster.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:39]: Verrazzano installation is optional

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+1. Expand **Advanced Settings** to make additional modifications to the default settings of your new cluster. 
+    * **YAML Manifests**: Specify post-provisioning installations.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:41]: Optionally supply additional YAML manifests that will be automatically installed after cluster creation. The total size of all additional YAML manifests may not exceed 500KB.

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+1. Copy or upload an SSH public key to manage authentication of the cluster.
+1. Configure the Control Plane. Select the **OCNE Version** first as it determines which Kubernetes versions are available, then choose a **Kubernetes Version** and a **Control Plane Shape**. You can leave the rest of the options at their default setting or modify them as needed.
+    
+    Under **Advanced**, you can choose to edit image tags for **ETCD**, **CoreDNS**, and **Calico**, or whether to install **OCI CCM/CSI** and **Calico**.
+1. Configure Node Pools, if necessary for your environment.
+1. Install Verrazzano on the cluster. Choose a **Verrazzano version** from the dropdown list. You can also expand **Advanced** to make changes to the Verrazzano Resource YAML. By default, Verrazzano is installed using the `managed-cluster` profile which enables a limited set of components on the cluster.
+1. Expand **Advanced Settings** to make additional modifications to the default settings of your new cluster. 
+    * **YAML Manifests**: Specify post-provisioning installations. 
+    * **Cluster Networking**: configure networking and proxy settings for the cluster.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:42]: Configure cluster IP Ranges and proxy settings

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    
+    Under **Advanced**, you can choose to edit image tags for **ETCD**, **CoreDNS**, and **Calico**, or whether to install **OCI CCM/CSI** and **Calico**.
+1. Configure Node Pools, if necessary for your environment.
+1. Install Verrazzano on the cluster. Choose a **Verrazzano version** from the dropdown list. You can also expand **Advanced** to make changes to the Verrazzano Resource YAML. By default, Verrazzano is installed using the `managed-cluster` profile which enables a limited set of components on the cluster.
+1. Expand **Advanced Settings** to make additional modifications to the default settings of your new cluster. 
+    * **YAML Manifests**: Specify post-provisioning installations. 
+    * **Cluster Networking**: configure networking and proxy settings for the cluster. 
+    * **Container Registry**: Specify a private registry for your container.
+1. Click **Create**. It will take a few minutes to provision all of the resources for your cluster.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:44]: up to 30 minutes for multi-node clusters

**Review by anders-swanson — COMMENTED** [review]
Was wondering if we should add some pictures? Thanks Robin!

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
+---
+title: Create Oracle Cloud Native Environment clusters running on OCI
+description: "Add OCNE managed clusters running on OCI to your multicluster environment"
```
Comment (prasad-shirodkar) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:3]: should this be self-managed instead?

File: content/en/docs/setup/provision-cluster/ocne-oci.md (hunk 5 above)
Comment (rhomast) [reply content/en/docs/setup/provision-cluster/ocne-oci.md:38]: @anders-swanson "..will allow pods to schedule '_what_?' on the control plane" 

and is there a 'not' missing from that sentence?

Comment (anders-swanson) [reply content/en/docs/setup/provision-cluster/ocne-oci.md:38]: I can explain over a call also

File: content/en/docs/setup/provision-cluster/ocne-oci.md (hunk 10 above)
Comment (anders-swanson) [reply content/en/docs/setup/provision-cluster/ocne-oci.md:3]: Context of control plane is self-managed, yes. I think we might want to say: "Create OCNE Clusters running on OCI from your Verrazzano instance"

Or something similar.

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 11]
```diff
@@ -27,21 +29,22 @@ To provision new Oracle Cloud Native Environment (OCNE) managed clusters on OCI,
 1. Expand **Labels and Annotations** to configure Kubernetes [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) and [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) for the cluster.
 1. Select the cloud credentials that you created. Ensure that the appropriate Region and Compartment are selected from their dropdown lists. 
 1. Click **Authenticate and Configure Cluster**.
-1. Set up your network. Choose **Quick Create** to build a new virtual cloud network (VCN) using default settings or **Existing Infrastructure** to use an already configured VCN. If you choose to use an existing VCN, make sure that it is configured to accept the [ports and protocols required by Kubernetes](https://kubernetes.io/docs/reference/networking/ports-and-protocols/).
+1. Set up your network. Select **Existing Infrastructure** to choose a virtual cloud network configured in your OCI account.
```
Comment (anders-swanson) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:32]: There is no **Existing Infrastructure** prompt anymore, you are taken straight to the VCN selection inputs

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+1. From the left menu, select **Clusters**, and then click **Create**.
+1. Select **Oracle OCNE on OCI** and provide a name for the cluster.
+1. Expand **Member Roles** to add any users that you want grant access to this cluster and their permissions.
+1. Expand **Labels and Annotations** to configure Kubernetes [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) and [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) for the cluster.
+1. Select the cloud credentials that you created. Ensure that the appropriate Region and Compartment are selected from their dropdown lists. 
+1. Click **Authenticate and Configure Cluster**.
+1. Set up your network. Choose the compartment where your VCN is located from the **VCN Compartment** dropdown list, then the VCN itself from the **Virtual Cloud Network** dropdown list. For **Cloud Plane Subnet**, **Load Balancer Subnet**, and **Worker Node Subnet**, choose subnets within the VCN.
```
Comment (rhomast) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:32]: @anders-swanson  Does the vcn compartment have to match the cloud credentials compartment?

Comment (rhomast) [reply content/en/docs/setup/provision-cluster/ocne-oci.md:32]: @theabmitra please see above question

Comment (theabmitra) [reply content/en/docs/setup/provision-cluster/ocne-oci.md:32]: Cloud credentials provide access to specific compartments based on how the user was setup on OCI console. So they are independent of each other .

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+description: "Add Oracle Cloud Native Environment self-managed clusters running on OCI to your multicluster environment"
+weight: 1
+draft: false
+---
+
+### Before you begin
+
+You'll need to:
+
+* [Enable the cluster API component]({{< relref "/docs/setup/modify-installation" >}})in the Verrazzano resource
```
[the remark this answers is not in the document]
Comment (theabmitra) [reply content/en/docs/setup/provision-cluster/ocne-oci.md:12]: Also we need to point to a page where how the component is enabled, not a general. like the component name is `clusterAPI`

File: content/en/docs/setup/provision-cluster/ocne-oci.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+* Generate an SSH key pair to use for cluster authentication
+
+### Create a new OCNE cluster on OCI 
+
+To provision new Oracle Cloud Native Environment (OCNE) managed clusters on OCI, complete the following steps:
+
+1. Log into the Rancher console. Find the Rancher console URL for your cluster at [Get console URLs]({{< relref "/docs/setup/access/console-urls.md" >}}).
+1. Open the navigation menu and select **Cluster Management**.
+1. From the left menu, select **Cloud Credentials**, and then click **Create**. Cloud credentials store the credentials for your cloud infrastructure provider.
+1. Choose **Oracle**.
```
Comment (theabmitra) [inline content/en/docs/setup/provision-cluster/ocne-oci.md:25]: This is `Oracle OCNE on OCI`

File: content/en/docs/setup/provision-cluster/_index.md [hunk 15]
```diff
@@ -0,0 +1,6 @@
+---
+title: Provision Additional Clusters
+description: "Learn how to add clusters to your environment using the console"
```
Comment (rosemarymarano) [inline content/en/docs/setup/provision-cluster/_index.md:3]: -> Add clusters to your environment using the console (IMO, most developers don't want to read about learning and prefer doing)

**Review by rosemarymarano — COMMENTED** [review]
One minor change suggested

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**